From d334a9783faf2a91b5c5b109948d27c97e727cd2 Mon Sep 17 00:00:00 2001
From: DHL <1435203275@qq.com>
Date: Wed, 18 Oct 2017 16:24:10 +0800
Subject: [PATCH 2/2] xiugai

---
 test1.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/test1.txt b/test1.txt
index 8970971..25e443e 100644
--- a/test1.txt
+++ b/test1.txt
@@ -1 +1,2 @@
-HelloWorld
\ No newline at end of file
+HelloWorld
+补丁内容：应该加一个“:”
\ No newline at end of file
-- 
1.9.2.msysgit.0

